js/src/api/transport/ws/ws.js

```diff
@@ -131,6 +131,8 @@ export default class Ws extends JsonRpcBase {
     this._autoConnect = true;
     this._retries = 0;
 
+    this.emit('open');
```

Comment:
It still have the isConnected & connected property. Basically when connected is set to true it should emit open, when set to false (errors), it should be emit false.